samples/window/main.cpp

```diff
@@ -207,6 +237,51 @@ int main() {
                 logger->info(keyEnumMap[event.key.code] + " is released");
             }
 
+            // Checking ButtonPressed events
+            if (event.type == lug::Window::Event::Type::ButtonPressed) {
+                logger->info(buttonEnumMap[event.button.code] + " is pressed at position {} {}", event.button.coord.x, event.button.coord.y);
+            }
+
+            // Checking ButtonReleased events
+            if (event.type == lug::Window::Event::Type::ButtonReleased) {
+                logger->info(buttonEnumMap[event.button.code] + " is released at position {} {}", event.button.coord.x, event.button.coord.y);
+            }
+            
+            // Checking MouseWheel events
+            if (event.type == lug::Window::Event::Type::MouseWheel) {
+                logger->info("Mouse scroll wheel offset {} {}", event.button.scrollOffset.xOffset, event.button.scrollOffset.yOffset);
+                if (event.button.ctrl) {
+                    logger->info("MODIFIER CTRL");
```

Comment:
Do we need this text? If yes, maybe we can change it to something more "esthetic".